server/tests/api_v4/test_enrollment.py

```diff
@@ -17,7 +20,34 @@
         "user",
     ),
 )
-async def test_device_enrollment(invitation: str, coolorg: CoolorgRpcClients):
+@pytest.mark.parametrize(
+    "issue7212",
+    (
+        False,
+        True,
+    ),
+    ids=("normal", "issue7212"),
+)
+async def test_device_enrollment(
+    invitation: str, coolorg: CoolorgRpcClients, issue7212: bool, monkeypatch
+):
+    if issue7212:
```

Comment:
would be better to call this flag with a more explicit name, and add a `# see issue https://github.com/Scille/parsec-cloud/issues/7212` comment

Reply:
[195d06b](https://github.com/Scille/parsec-cloud/pull/7307/commits/195d06bc98e169d4a6f8583ab5f38c1bd1ef9de6)